public/media-hunt/API/discover.js

```diff
@@ -0,0 +1,9 @@
+const ROOT = {
+  get: {
+    discover: {
+      url: "/discover/movie",
+      url: "/discover/tv"
```

Comment:
Object can't have two keys with same name. VsCode has probably highlighted that as well with red underline. Please name them movie and tv appropriately.

Reply:
got it, will be adjusted with the next commit.

Reply:
added genres, top rated and reviews to movie and tv, and discover.js file removed, and placed in movie and tv js files.
Changes are commited.

Reply:
i noticed that from API/index.js discover file needs to be removed. I will remove it.

Reply:
> 

discover file is removed from index.js